File: src/main/java/io/papermc/voidWorld/mobs/MobEquipment.java
package io.papermc.voidWorld.mobs;

import org.bukkit.inventory.ItemStack;

public record MobEquipment(
        ItemStack mainHand,
        ItemStack offHand,
        ItemStack helmet,
        ItemStack chestplate,
        ItemStack leggings,
        ItemStack boots
) {}

File: src/main/java/io/papermc/voidWorld/mobs/helper/MobEquipment.kt
package io.papermc.voidWorld.mobs.helper

@JvmRecord
data class MobEquipment(
    val mainHand: ItemStackConfiguration?,
    val offHand: ItemStackConfiguration?,
    val helmet: ItemStackConfiguration?,
    val chestplate: ItemStackConfiguration?,
    val leggings: ItemStackConfiguration?,
    val boots: ItemStackConfiguration?
) 